docker/publish.sh

```diff
@@ -12,40 +15,50 @@ pip3 install requests beautifulsoup4
 
 function export_geosupport_versions {
     VERSIONSTRING=$(python3 geosupport_versions.py)
-
+    echo $VERSIONSTRING
     export $(echo "$VERSIONSTRING" | sed 's/#.*//g' | xargs)
     export VERSION=$MAJOR.$MINOR.$PATCH
 
     export BUILD_ARGS="--build-arg RELEASE=$RELEASE --build-arg MAJOR=$MAJOR --build-arg MINOR=$MINOR --build-arg PATCH=$PATCH"
 }
 
 function common {
-    DOCKER_IMAGE_NAME=nycplanning/$1
```

Comment:
This is a function rather than just called because it will run into errors if an invalid image name has been specified. So in the section where we match image name (line 44), an error we define gets thrown rather than something more confusing